javascript/rename_models_in_charts.js

```diff
@@ -146,29 +146,33 @@ const rename = async () => {
     console.log('This project has '+spaceResponse.results.length+' spaces' )
 
     spaceResponse.results.map(async (spaceSummary) => {
-        
-            if (debug) console.log(`Making request to get space ${spaceSummary.uuid}`)
+        if (debug) console.log(`Making request to get space ${spaceSummary.uuid}`)
+
+        const spaceFetch = await fetch(`${apiUrl}/projects/${projectUuid}/spaces/${spaceSummary.uuid}`, {
+            method: 'GET',
+            headers,
+        });
+        if (debug) console.log('Space response', spaceFetch.status)
 
-            const spaceFetch = await fetch(`${apiUrl}/projects/${projectUuid}/spaces/${spaceSummary.uuid}`, {
-                method: 'GET',
-                headers,
-            });
-            if (debug) console.log('Space response', spaceFetch.status)
+        const space = (await spaceFetch.json()).results
+        console.log(`Space ${space.name} has ${space.queries.length} charts and ${space.queries.dashboards} dashboards` )
 
-            const space = (await spaceFetch.json()).results
-            console.log(`Space ${space.name} has ${space.queries.length} charts and ${space.queries.dashboards} dashboards` )
+        space.queries.map(async (query) => {
+            if (debug) console.log(`Making request to get chart ${query.uuid}`)
+            let savedChart = undefined
+            let savedChartResponse = undefined
```

Comment:
these are the main changes. defining variables out of the try scope so catch block can see them